packages/gestalt/src/Badge.js

```diff
@@ -50,10 +67,60 @@ export default function Badge({ position = 'middle', text, type = 'info' }: Prop
     'lightWash': 'washLight',
   };
 
-  const cs = cx(styles.Badge, styles[position], colors[TYPE_COLOR_MAP[type]], {
+  const inBadgeExp = useInExperiment({
+    webExperimentName: 'web_gestalt_redesigned_badge',
+    mwebExperimentName: 'mweb_gestalt_redesigned_badge',
+  });
+
+  const csBadge = cx(styles.Badge, styles[position], colors[TYPE_COLOR_MAP[type]], {
     [styles.darkWash]: type === 'darkWash',
     [styles.lightWash]: type === 'lightWash',
   });
 
-  return <div className={cs}>{text}</div>;
+  const csNewBadge = cx(styles.NewBadge, styles[position], {
+    [styles.darkWash]: type === 'darkWash',
+    [styles.lightWash]: type === 'lightWash',
+    [styles.info]: type === 'info' && !tooltip?.text,
+    [styles.interactiveInfo]: type === 'info' && tooltip?.text,
+    [colors.errorBase]: type === 'error',
+    [colors.warningBase]: type === 'warning',
+    [colors.successBase]: type === 'success',
+    [colors.tertiary]: type === 'neutral',
+    [colors.recommendationBase]: type === 'recommendation',
+    [colors.washDark]: type === 'darkWash',
+    [colors.washLight]: type === 'lightWash',
+  });
+
+  if (inBadgeExp) {
+    const badgeComponent = (
+      <div className={csNewBadge}>
+        <Flex alignItems="center" gap={1}>
+          {type === 'info' && tooltip?.text ? (
```

Comment:
nit: we should be able to make this check once and store it as `shouldUseTooltip` or something so this logic isn't repeated

Reply:
DONE

keeping last one for Flow

Reply:
DONE